OCP-4.X/roles/install/tasks/main.yml

```diff
@@ -122,16 +121,22 @@
         cd {{ workdir }}
         export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE={{ OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE }}
         export _OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE={{ OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE }}
-        bin/openshift-install create cluster --log-level=debug 2>&1 | tee -a {{ OPENSHIFT_INSTALL_LOG_DIR }}/install.log
+        bin/openshift-install create cluster --log-level=debug 2>&1 | tee -a {{ OPENSHIFT_INSTALL_LOG_DIR }}/install.stdout.log
```

Comment:
We can go ahead and get rid of the entire "tee the output to a log file" idea I originally had, it failed when the installer decided the actual log it records and stdout/stderr should be different.

Reply:
Done.